lms/static/sass/discussion/elements/_actions.scss

```diff
@@ -58,7 +58,7 @@
       &:after,
       &:before {
         bottom: 100%;
-        @include right(3px);
+        right: 6px;
```

Comment:
Don't freak out! In this case, the way the dropdown is appearing, we want the caret/nob to be in the same place on both LTR and RTL. This may change after I refactor the code in the next set of work.